### DISCUSSION TITLE
模板输出样例展示（完整版）

### DISCUSSION THREAD

**Discussion by skyleaworlder** [TJ-CSCCG/TongjiThesis#21] [General]
## 模板输出样例展示（v1.3.1）

以下展示模板在默认配置下的编译输出效果，共 40 页。完整 PDF 可从 [Release](https://github.com/TJ-CSCCG/TongjiThesis/releases) 页面下载，也可通过 [Overleaf 模板](https://www.overleaf.com/latex/templates/tongji-university-undergraduate-thesis-template/tfvdvyggqybn) 在线预览。

模板涵盖封面、信息说明页、中英文摘要、目录、正文（模板使用指南、绪论、图表与代码、数学公式与符号、引用与注释、总结与展望）、参考文献、附录（自动「附录」起始页 + A./A.1 节编号）和谢辞等内容。

**Comment by skyleaworlder** [comment]
![Page 1](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0001.jpg)
![Page 2](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0002.jpg)
![Page 3](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0003.jpg)
![Page 4](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0004.jpg)
![Page 5](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0005.jpg)
![Page 6](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0006.jpg)
![Page 7](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0007.jpg)
![Page 8](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0008.jpg)
![Page 9](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0009.jpg)
![Page 10](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0010.jpg)
![Page 11](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0011.jpg)
![Page 12](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0012.jpg)
![Page 13](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0013.jpg)
![Page 14](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0014.jpg)
![Page 15](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0015.jpg)
![Page 16](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0016.jpg)
![Page 17](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0017.jpg)
![Page 18](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0018.jpg)
![Page 19](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0019.jpg)
![Page 20](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0020.jpg)
![Page 21](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0021.jpg)
![Page 22](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0022.jpg)
![Page 23](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0023.jpg)
![Page 24](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0024.jpg)
![Page 25](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0025.jpg)
![Page 26](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0026.jpg)
![Page 27](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0027.jpg)
![Page 28](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0028.jpg)
![Page 29](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0029.jpg)
![Page 30](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0030.jpg)
![Page 31](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0031.jpg)
![Page 32](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0032.jpg)
![Page 33](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0033.jpg)
![Page 34](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0034.jpg)
![Page 35](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0035.jpg)
![Page 36](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0036.jpg)
![Page 37](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0037.jpg)
![Page 38](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0038.jpg)
![Page 39](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0039.jpg)
![Page 40](https://media.githubusercontent.com/media/TJ-CSCCG/TJCS-Images/TongjiThesis/preview/main_page-0040.jpg)

**Reply by nanxi666** [reply]
好像我这边加载出来不是完整版？

**Reply by admin** [reply]
> 好像我这边加载出来不是完整版？

你好，现在有了。
